src/genie/libs/parser/iosxe/tests/ShowApDot115GhzChannel/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,28 @@
+expected_output = {
+    "channel_assignment": {
+        "chan_assn_mode": "AUTO",
+        "chan_upd_int": 12,
```

Comment:
better to have unit in key name like `chan_upd_int_hours` or get the value as string with unit.
```
"chan_upd_int": "12 Hours",

Reply:
updated to include the value as a string since you can also have interval of "minutes"

Reply:
confirmed